transport/http/constants.go

```diff
@@ -20,8 +20,18 @@
 
 package http
 
-// ProcedureHeader is the header used by the HTTP outbound to send and receive
-// the procedure name.
-const ProcedureHeader = "YARPC-Procedure"
+const (
+	// CallerHeader is the HTTP header used to indiate the service doing the calling
+	CallerHeader = "YARPC-Caller"
+
+	// TTLMSHeader is the HTTP header used to indicate the ttl in ms
+	TTLMSHeader = "YARPC-TTLms"
```

Comment:
@kriskowal I'm ok with either - let's settle on something and do that for all langs. Also, I think you want camel casing, not screaming hyphen case :)